src/pages/CON_DeleteContactOverride.page

```diff
@@ -2,15 +2,22 @@
     <apex:slds />
     <apex:form styleClass="slds">
         <apex:outputPanel rendered="{!sforceNav == null}">
-            <c:UTIL_PageMessages />
-            <div class="slds-p-around_medium">
-                <apex:outputText value="{!helpDeleteAccountInstead}" rendered="{!NOT(shouldDeleteContactAlone)}" styleClass="slds-text-body_regular"/>
-            </div>
-            <div class="slds-button-group">
-                <apex:outputLink target="_self" title="{!$Label.REL_Return_to_Contact}" value="/{!$CurrentPage.parameters.id}" styleClass="slds-button slds-button_brand">
+            <!-- PAGE HEADER -->
+            <c:UTIL_PageHeader headerLabel="{!$ObjectType.Contact.Label}"
+                               header="{!Contact.Name}"
+                               icon="contact_120" iconCategory="standard"
+                               showSaveBtn="false" cancelLabel="{!$Label.bdiBtnClose}"
+                               cancelAction="{!getRedirect}" cancelReRender="vfForm"/>
+            
+            <!-- PAGE MESSAGE -->
+            <c:UTIL_PageMessages allowClose="false"/>
+            
+            <div class="slds-p-around_small">
+                <apex:commandButton value="{!$Label.Delete_Contact_Leave_Account}" action="{!deleteContactOnly}" styleClass="slds-button slds-button_neutral"/>
```

Comment:
@bethbrains the label here is using underscores but it looks like the label that was added to the package does not "DeleteContactLeaveAccount".